### ISSUE TITLE
Assign expanded iPad layouts for devices that are wide enough

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-iOS#352]
### Terms

- [X] I have searched [open and closed design issues](https://github.com/scribe-org/Scribe-iOS/issues?q=is%3Aissue+label%3Adesign)
- [X] I agree to follow Scribe-iOS' [Code of Conduct](https://github.com/scribe-org/Scribe-iOS/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue would be to assign the keyboard layouts that were made in #33 to iPads that have wide enough screens. Ideally there would also be some work done to space the keys appropriately, but the team can also assist with this 😊

### Contribution

Happy to work on this myself or assist someone who has interest in working on it!

**Comment by henrikth93** [comment]
I am interested in doing this

**Comment by andrewtavis** [comment]
Sounds great, @henrikth93! Glad to have some help on this 😊 Assigning you now, and please let us know if you need some assistance!

**Comment by andrewtavis** [comment]
This issue was closed by 2b544e0 😊😊 Here's a quick summary of what I did, @henrikth93:

- Basic formatting and fixing the comments
- The condition to check if a keyboard should be expanded was just outside of `isFirstKeyboardLoad`, so I put it above `conditionallyShowTopNumbersRow` and a few other baseline functions
- The check for the home button wasn't working, so it's now just checking for a width > 768
  - I feel like we should do the menu option for this before release so the person has the option to always turn it off or on, and in this way it should generally work :)
- I like how you were checking for the `symbolKeys` for widths and changed it so that we always do this instead of the number keys that might not be there 👍👍
- I added `StackViewNum` into some of the steps that are adding keys (this is what was causing the top row to not show up as it never was having the keys added to it)
- Unrelated bug fix for how the menu option to add a period and comma to the letter keys was being implemented as this was actually replacing the bottom row in cases when it shouldn't have been 😊
- Random minor fixes that I picked up along the way :)

Again very sorry for the wait on this! Let me know if you have interest in working on #362. I'm also going to be making an issue now to add the functionality into keys like the tab key and other new ones that we have on the expanded layouts.
